tests/kernels/test_cache.py

```diff
@@ -177,16 +179,14 @@ def test_swap_blocks(
     num_blocks: int,
     dtype: torch.dtype,
     seed: int,
-    device: int,
+    device: str,
 ) -> None:
     random.seed(seed)
     torch.random.manual_seed(seed)
     if torch.cuda.is_available():
         torch.cuda.manual_seed(seed)
-    src_device = f"{direction[0]}:{device}" if direction[
-        0] == "cuda" else direction[0]
-    dst_device = f"{direction[1]}:{device}" if direction[
-        1] == "cuda" else direction[1]
+    src_device = device if direction[0] == "cuda" else "cpu"
```

Comment:
Are these change necessary?

Reply:
This is a unit test bug. See this PR [https://github.com/vllm-project/vllm/pull/2750](url).

Reply:
Got it